.github/workflows/ci.yaml

```diff
@@ -15,20 +18,12 @@ jobs:
       uses: actions/setup-go@v2
       with:
         go-version: '1.17'
-    - name: Install dependencies
+    - name: Run tests
+      run: go test .
+    - name: Build
       run: |
-        go version
-        go get -u golang.org/x/lint/golint
-    - name: Run cross build
-      run: script/build.sh
-    # Run vet on the code
-    - name: Run vet
-      run: go vet .
```

Comment:
It looks like we've lost the `go vet` step here - did you conclude that this is unnecessary now that we are actually running tests?!

Reply:
Thanks - have re-added.